VictoriaAlbert/DetailViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  DetailViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Eashir Arafat on 11/14/16.
+//  Copyright © 2016 Evan. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+    var ring: Ring?
+    @IBOutlet weak var ringImageView: UIImageView!
+    @IBOutlet weak var museumNumberLabel: UILabel!
+    
+    @IBOutlet weak var locationTextView: UITextView!
+   
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        loadData()
+        
+        
+        // Do any additional setup after loading the view.
+    }
+    
+    func loadData() {
+        var firstSixLetters = [Character]()
+        var emptyString = ""
+        for eachCharacter in ((ring?.imageID)?.characters)! {
+            
+            if firstSixLetters.count != 6 {
```

Comment:
This is one way to do it but it might have been more efficient (to understand) to use the prefix method.